### PR TITLE
Add custom MFA error page

### PR DESCRIPTION
**Pull request by pablothedude**
**Be aware this PR is build on top on #864 so this PR needs to be reviewed first**

Add a custom error page when the response from configured MFA IdP is
invalid.

https://www.pivotaltracker.com/story/show/172718794

### CODE REVIEW

**Comment by pablothedude** [comment]
@thijskh could you verify the translations added in this PR? I've changed the English translation slightly to avoid possession rules with the apostrophe in `%organizationNoun%'s`.

File: languages/messages.nl.php [hunk 1]
```diff
@@ -229,6 +229,8 @@
     'error_no_authentication_request_received' => 'Fout - Geen authenticatie-aanvraag ontvangen.',
     'error_authn_context_class_ref_blacklisted' => 'Fout - Waarde van AuthnContextClassRef is niet toegestaan',
     'error_authn_context_class_ref_blacklisted_desc' => '<p>Je kunt niet inloggen omdat je %organisationNoun% een waarde stuurde voor AuthnContextClassRef die niet is toegestaan.</p>',
+    'error_invalid_mfa_authn_context_class_ref' => 'Fout - Multi factor authenticatie muslikt',
```
Comment (thijskh) [inline languages/messages.nl.php:232]: Qua avian virussen en zoönoserisico in deze tijden misschien beter tijdelijk geen mussen te likken? :zany_face: 

Liever: multi-factor authenticatie met koppelteken

File: languages/messages.nl.php [hunk 2]
```diff
@@ -229,6 +229,8 @@
...
+    'error_invalid_mfa_authn_context_class_ref_desc' => '<p>Jouw %organisationNoun% dwingt geavanceerde multi-factor authenticatie af voor deze dienst. Maar je second factor kon niet gevalideerd worden. Neem hiervoor contact op met de service desk van je %organisationNoun% om dit op te lossen.</p>',
```
Comment (thijskh) [inline languages/messages.nl.php:233]: Aantal dingen aangepast:

"Jouw %organisationNoun% vereist multi-factor authenticatie deze dienst. Je tweede factor kon echter niet gevalideerd worden. Neem contact op met de servicedesk van je %organisationNoun% om dit op te lossen."

**Comment by pablothedude** [comment]
Bedankt voor de feedback, de aanpassingen zijn verwerkt.

**Comment by pablothedude** [comment]
@domgon I want to let you know we added some translations. Would you be so kind to review the PT translations in this PR? Thanks!

**Comment by domgon** [comment]
Hello @pablothedude ,

I'm checking the translations.

I will PR when possible

Best Regards

**Comment by domgon** [comment]
Hi, you can find the translations in Portuguese below for the specific variables. Even so, I'm gonna perform a PR of the entire **messages.pt.php** according to the **messages.en.php** changed yesterday.

```
    'error_invalid_mfa_authn_context_class_ref' => 'Erro - falha na autenticação segundo fator de autenticação (2FA)',
    'error_invalid_mfa_authn_context_class_ref_desc' => '<p>A sua %organisationNoun% requer segurança adicional para este serviço, por meio de um segundo fator de autenticação (2FA). No entanto, o seu segundo fator de autenticação não pôde ser verificado. Entre em contato com o suporte da sua %organisationNoun% para validar esta situação.</p>',
```
Best regards,
Domingos Gonçalves

**Comment by pablothedude** [comment]
Muito obrigado @domgon, I'll add the translations to this PR.

**Pull request merged**